dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionGroupRestControllerIT.java

```diff
@@ -763,11 +879,12 @@ public void deleteCollectionSubmittersGroupTestParentCommunityAdmin() throws Exc
     }
 
     // This is currently not supported in DSpace API
```

Comment:
Again, please remove this comment as it's no longer valid after the `@Ignore` was removed.